api/src/main/resources/application.yaml

```diff
@@ -9,6 +9,8 @@ spring:
 
 
 pharmagator:
+  drlz-registry:
```

Comment:
I do not see where you use this property of the file

Reply:
I was planning to use this property later in code. I'll delete it for now.